clients/venice-push-job/src/main/java/com/linkedin/venice/input/kafka/KafkaInputFormat.java

```diff
@@ -1,8 +1,8 @@
-package com.linkedin.venice.hadoop.input.kafka;
+package com.linkedin.venice.input.kafka;
```

Comment:
Would prefer keeping this in MR-specific packages. Is it possible?

Reply:
Sure, I imagined this is not MR specific, I'll put it back, or split it into common and MR specific bits.

Reply:
moved them back.